.gitignore

```diff
@@ -0,0 +1,9 @@
+# Ignore backup files created by editors like emacs.
+*~
+# Ignore cmake files.
+*CMakeCache.txt
+*CMakeFiles
+# Ignore tmp directory/file. We use it sometimes to write debug information.
+tmp
+# Ignore build trees.
+build-*
```

Comment:
Shrinidhi, you misunderstood my comment. I'm asking about the hyphen "-". I commonly use a build directory called just `build`, without the hyphen. I have also seen `build_`. So I'm asking whether it would be better to use the pattern `+build*` rather than `+build-*`.